api/internal/context121/go121_context_compat.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2014 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+package context121
```

Comment:
If you move this to the project root, as in teleport/internal/context121, then I believe it can be imported by both lib/ and api/ (thus no need for 2 copies).

Reply:
Oh but nevermind, as this makes api/ depend on teleport/. 😢